src/main/java/it/pagopa/pn/paperchannel/model/Range.java

```diff
@@ -0,0 +1,18 @@
+package it.pagopa.pn.paperchannel.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import java.math.BigDecimal;
+
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+public class Range {
```

Comment:
In realtà anche questa è una entità, sebbene non mappi una tabella (quindi va annotata con @DynamoDbBean). La chiamerei quindi PnRange e la inserirei nel package entities.

Reply:
Le modifiche richieste sono state effettuate.